my_work_R/my_happy_script.R

```diff
@@ -0,0 +1,94 @@
+
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+project_output_dir <- 'my_dir'
+
+dir.create(project_output_dir)
+
+mendota_file <- file.path(project_output_dir, 'model_RMSEs.csv')
+item_file_download('5d925066e4b0c4f70d0d0599', names = 'me_RMSE.csv', destinations = mendota_file, overwrite_file = TRUE)
+
+eval_data <- readr::read_csv(mendota_file, col_types = 'iccd') %>%
+  filter(str_detect(exper_id, 'similar_[0-9]+')) %>%
+  mutate(col = case_when(
+    model_type == 'pb' ~ '#1b9e77',
+    model_type == 'dl' ~'#d95f02',
+    model_type == 'pgdl' ~ '#7570b3'
+  ), pch = case_when(
+    model_type == 'pb' ~ 21,
+    model_type == 'dl' ~ 22,
+    model_type == 'pgdl' ~ 23
+  ), n_prof = as.numeric(str_extract(exper_id, '[0-9]+')))
+
+
+
+png(file = file.path(project_output_dir, 'figure_1.png'), width = 8, height = 10, res = 200, units = 'in')
+par(omi = c(0,0,0.05,0.05), mai = c(1,1,0,0), las = 1, mgp = c(2,.5,0), cex = 1.5)
+
+plot(NA, NA, xlim = c(2, 1000), ylim = c(4.7, 0.75),
+     ylab = "Test RMSE (°C)", xlab = "Training temperature profiles (#)", log = 'x', axes = FALSE)
+
+n_profs <- c(2, 10, 50, 100, 500, 980)
+
+axis(1, at = c(-100, n_profs, 1e10), labels = c("", n_profs, ""), tck = -0.01)
+axis(2, at = seq(0,10), las = 1, tck = -0.01)
+
+# slight horizontal offsets so the markers don't overlap:
+offsets <- data.frame(pgdl = c(0.15, 0.5, 3, 7, 20, 30)) %>%
+  mutate(dl = -pgdl, pb = 0, n_prof = n_profs)
```

Comment:
We should really convert this to ggplot2.

Reply:
Noted in #20